src/new-client/src/app/site/[site]/users/page.test.tsx

```diff
@@ -8,22 +7,22 @@ import { fetchUsers } from '../../../lib/users';
 jest.mock('../../../lib/roles');
 jest.mock('../../../lib/users');
 
-const fetchUsersMock = fetchUsers as jest.Mock<any>;
-const fetchRolesMock = fetchRoles as jest.Mock<any>;
+const fetchUsersMock = fetchUsers as jest.Mock<Promise<User[]>>;
+const fetchRolesMock = fetchRoles as jest.Mock<Promise<Role[]>>;
 
 describe('<UserManagement />', () => {
   it('renders', async () => {
     fetchUsersMock.mockResolvedValue(mockUserAssignments);
     fetchRolesMock.mockResolvedValue(mockRoles);
 
     const jsx = await UsersPage({ params: { site: mockSiteId } });
-    await render(jsx);
+    render(jsx);
+
+    expect(screen.getByText('Manage Staff Roles')).toBeVisible();
 
     expect(
-      screen.getByRole('table', {
-        name: "Manage your current site's staff roles",
-      }),
-    ).toBeInTheDocument();
+      screen.getByText("Manage your current site's staff roles"),
```

Comment:
One to ask the design team I think, but I do wonder if the UX might be clearer if this was `Manage staff roles at {siteDisplayName}` - perhaps an improvement for later down the road

Reply:
I think that once the site information is part of the general layout it wouldn't  be necessary to repeat that information, so I think it is ok to leave as is.